### ISSUE TITLE
Not an issue but a request for help

### ISSUE THREAD

**Issue by wisdomlight** [mattydebie/bitwarden-rofi#38]

**Comment by wisdomlight** [comment]
Hi folk
would someone mind explaining to me how to install Rofi on linux mint?
I downloaded the zip 
extracted it
put all the files in the ~/bin
My ~/bin is in my $PATH and looks like this:

```
sherab@sherab-Librem-13-v2:~/bin$ ls
bwmenu  img  lib-bwmenu  LICENSE  README.md
```

but i get the following error:
```
sherab@sherab-Librem-13-v2:~/bin$ bwmenu 
/home/sherab/bin/bwmenu: line 52: rofi: command not found
/home/sherab/bin/bwmenu: line 86: rofi: command not found
```

Any suggestions?
I am sure i miss something!

**Comment by Mange** [comment]
Have you installed `rofi`? It should be in your package manager.
(This should probably be mentioned in the README too.)

You only need to place the `bwmenu` and `lib-bwmenu` files inside your `~/bin`.

**Comment by wisdomlight** [comment]
Thank you @Mange 
I didn't get that Rofi is what needs to be installed from the Software manager.
So now when i run it do i just need to put the credentials of BW ?

**Comment by Mange** [comment]
Yeah, follow the install instructions of bwcli and log in to your account using it. After that you should be able to use bwmenu.
